plugins/dotnet-auth-core-identity/src/core/create-method-authorize-annotation.ts

```diff
@@ -0,0 +1,40 @@
+import { Entity, EnumEntityAction } from "@amplication/code-gen-types";
+import { CsharpSupport, Method } from "@amplication/csharp-ast/src";
+import { getEntityRoleMap } from "./get-entity-roles-map";
+
+export function createMethodAuthorizeAnnotation(
+  method: Method,
+  roles: string
+): void {
+  roles &&
+    method.annotations?.push(
```

Comment:
you check if `annotations` exist (with the `?`) - but if it doesn't, you still need to create it and add this annotation